packages/api/src/Api.ts

```diff
@@ -116,6 +116,7 @@ export class Api extends ApiPromise {
 
     decorateCennznetExtrinsics(): void {
         decorateExtrinsics(this);
+        decorateExtrinsicsSubmittables(this);
```

Comment:
can you combine them together? decorateExtrinsics and decorateExtrinsicsSubmittables

Reply:
yes